### PR TITLE
ORDERS-5733: [add] Order Fees to Refund documentation

### PR DESCRIPTION
**Pull request by jpajar**
# [
[ORDERS-5733](https://bigcommercecloud.atlassian.net/browse/ORDERS-5733)
## What changed?

* Added `fee` item type to refund API documentation

## Release notes draft

* 

## Proof
image

[ORDERS-5733]: https://bigcommercecloud.atlassian.net/browse/ORDERS-5733?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/orders.v3.yml [hunk 1]
```diff
@@ -2635,6 +2639,30 @@ components:
           minLength: 0
           maxLength: 255
           example: 'Service fee'
+    FeeItem:
+      type: object
+      title: Fee
+      description: 'Use this to refund a custom value at the order level.'
```
Comment (bc-vincent-zhao) [inline reference/orders.v3.yml:2645]:
```suggestion
      description: 'Use this to refund a custom fee at the order level.'
```

**Review by bc-vincent-zhao — DISMISSED** [review]

**Review by bc-luke — DISMISSED** [review]

File: reference/orders.v3.yml [hunk 2]
```diff
@@ -2635,6 +2639,30 @@ components:
           minLength: 0
           maxLength: 255
           example: 'Service fee'
+    FeeItem:
+      type: object
+      title: Fee
+      description: 'Use this to refund a custom fee at the order level.'
```
Comment (bc-traciporter) [inline reference/orders.v3.yml:2645]:
```suggestion
      description: 'Use this field to refund a custom fee at the order level.'
```

**Review by bc-traciporter — APPROVED** [review]
LGTM!  Approved

**Review by bc-vincent-zhao — APPROVED** [review]

**Comment by bc-traciporter** [comment]
Hi @jpajar, @bc-vincent-zhao and @WillemHoman 
Can I add this change to the release notes tomorrow? How does this sound?

We're happy to announce that we've added a fee item type to [Refund](https://developer.bigcommerce.com/docs/rest-management/transactions/payment-actions#create-a-refund-quote) endpoints for the Orders API to represent order level fees like package insurance, gratuities/tips, and credit card fees. To see an example request, see [Creating refund](https://developer.bigcommerce.com/docs/store-operations/orders/refunds#creating-refund-quotes) quotes.

**Pull request merged**